aries-site/src/components/seo/Meta.js

```diff
@@ -92,10 +92,10 @@ export const Meta = ({ title, description, canonicalUrl, socialImageUrl }) => {
       <meta
         httpEquiv="Content-Security-Policy"
         content="default-src 'self' 'unsafe-eval'; 
-        style-src 'self' *.hpe.com/hfws-static/slim/css/ 'unsafe-inline';
-        connect-src 'self' *.githubusercontent.com/grommet/hpe-design-system/ https://www.google-analytics.com https://www.github.com/grommet/ https://eyes.applitools.com;
+        style-src 'self' *.hpe.com/hfws-static/5/css/ 'unsafe-inline';
+        connect-src 'self' *.githubusercontent.com/grommet/hpe-design-system/ https://www.google-analytics.com https://www.github.com/grommet/ https://eyes.applitools.com *.hpe.com/hpe/api/;
```

Comment:
assuming the `*.hpe.com/hpe/api` is for any of the auth / signin, correct?

Reply:
Yes although we have the sign in set to off but it was still trying to hit the user api.